staker/block_validator.go

```diff
@@ -1060,6 +1065,9 @@ func (v *BlockValidator) Initialize(ctx context.Context) error {
 		}
 	}
 	log.Info("BlockValidator initialized", "current", v.currentWasmModuleRoot, "pending", v.pendingWasmModuleRoot)
+	if err := v.StatelessBlockValidator.Initialize([]common.Hash{v.currentWasmModuleRoot, v.pendingWasmModuleRoot}); err != nil {
```

Comment:
check if they are identical, and if so only send one - this is common and we don't want a warning on boot

Reply:
Done..